### DISCUSSION TITLE
can't import spark_base in 3.9.0 for array comparisons

### DISCUSSION THREAD

**Discussion by cvandenende** [moj-analytical-services/splink#1276] [General]
Hi all,

We have written some custom array comparisons, and since the latest update (3.9.0) the import:
`
from splink.spark import spark_base as spark_base 
`
No longer imports the spark_base functions and our array comparisons don't work. Just wanted to let you know :)

**Comment by ADBond** [comment]
Apologies - we changed some of the folder structure around comparisons in `3.9.0` (see #1248 for details). The comparison library and comparison level library functions should still be importable from the same place, but all of the `...Base` objects and other helper functions have been moved under the [`splink.spark.spark_helpers`](https://github.com/moj-analytical-services/splink/tree/master/splink/spark/spark_helpers) module.

You should be able to use instead: `from splink.spark.spark_helpers import spark_base as spark_base`

**Reply by ADBond** [reply]
Also if you think any of your custom array comparisons might be more useful generally you can always put in a PR to add them to Splink's existing library 😄

**Reply by cvandenende** [reply]
Thank you so much for the clarification. At the moment we are still developing the code, and it is a fairly niche use-case, but we will consider putting in a PR once we get it running :)
